services/postgres/schema.prisma

```diff
@@ -0,0 +1,136 @@
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL") 
+
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model Org {
+  id               String          @id @unique @default(cuid())
+  users            User[]
+  apps             App[]
+  active           Boolean         @default(true)
+  deleted          Boolean         @default(false)
+  createdAt        DateTime
+  updatedAt        DateTime
+}
+
+model Tenant {
+  id        String   @id @unique @default(cuid())
+  active    Boolean  @default(true)
+  paymentLedger    PaymentLedger?
+  relayLedger RelayLedger?
+  keys Key[]
+  deleted   Boolean  @default(false)
+  createdAt DateTime
+  updatedAt DateTime
+}
+
+model User {
+  id        String   @id @unique @default(cuid())
+  org       Org      @relation(fields: [orgId], references: [id])
+  orgId     String
+  apps      App[]    @relation("UserApps")
+  active    Boolean  @default(true)
+  deleted   Boolean  @default(false)
+  createdAt DateTime
+  updatedAt DateTime
+}
+
+model App {
+  id        String    @id @unique @default(cuid())
+  org       Org       @relation(fields: [orgId], references: [id])
+  orgId     String
+  user      User      @relation("UserApps", fields: [userId], references: [id])
+  userId    String
+  appRules  AppRule[]
+  key String // api-key
+  active    Boolean   @default(true)
+  deleted   Boolean   @default(false)
+  createdAt DateTime
+  updatedAt DateTime
+}
+
+model AppRule {
+  id        String   @id @unique @default(cuid())
+  app       App      @relation(fields: [appId], references: [id])
+  appId     String
+  rule      RuleType?
+  value String
+  active    Boolean  @default(true)
+  deleted   Boolean  @default(false)
+  createdAt DateTime
+  updatedAt DateTime
+}
+
+model RuleType {
+  id              String   @id @unique @default(cuid())
+  name            String
+  isEditable      Boolean
+  isMultiple      Boolean
+  validationType  String
+  validationValue String
+  appRule   AppRule  @relation(fields: [appRuleId], references: [id])
+  appRuleId String          @unique
+  active          Boolean  @default(true)
+  deleted         Boolean  @default(false)
+  createdAt       DateTime
+  updatedAt       DateTime
+}
+
+model Key {
```

Comment:
I'm thinking "Key" might be too generic here, ApiKey or TenantAuthKey might be more descriptive.

Reply:
TenantAuthKey sounds best to me